storage/pebble/transactions.go

```diff
@@ -46,5 +47,12 @@ func (t *Transactions) Get(ID common.Hash) (models.Transaction, error) {
 		return nil, err
 	}
 
-	return models.UnmarshalTransaction(val)
+	heightVal, err := t.store.get(latestCadenceHeightKey)
+	if err != nil {
+		heightVal = []byte{0, 0, 0, 0, 0, 0, 0, 0}
```

Comment:
In case of `err`, I am just adding a default value of `0`. Not sure how to handle that.